.github/CODEOWNERS

```diff
@@ -1 +1,11 @@
-* @roluk @diegopinate @CalebGerman
+# Global owners
+* @diegopinate @CalebGerman @hnn0003 @sarah-pontier 
```

Comment:
I wouldn't mind this one to just have global owners

Reply:
Should be changed. :)